jans-cedarling/cedarling/src/init/jwt_algorithm.rs

```diff
@@ -5,11 +5,9 @@
  * Copyright (c) 2024, Gluu, Inc.
  */
 use crate::bootstrap_config::BootstrapConfig;
-use crate::jwt::{string_to_alg, Algorithm, ParseAlgorithmError};
+use crate::jwt::{self, string_to_alg, Algorithm};
 
-pub(crate) fn parse_jwt_algorithms(
-    config: &BootstrapConfig,
-) -> Result<Vec<Algorithm>, ParseAlgorithmError> {
+pub(crate) fn parse_jwt_algorithms(config: &BootstrapConfig) -> Result<Vec<Algorithm>, jwt::Error> {
```

Comment:
Why here we return all possible errors in JWT module if can happen only one?

Reply:
i'll fix it in #9945 if that's okay. #9870 pulled all the logic out of the `JwtService` which made a mess.

Reply:
actually, i changed my mind and just tried to manually revert the changes here: [c8c8945](https://github.com/JanssenProject/jans/pull/9939/commits/c8c8945883b97b1041fccbba0c10f3c9b17c4d19). should be back to it's original implementation.